spec/requests/api/tenant_quotas_spec.rb

```diff
@@ -0,0 +1,246 @@
+describe "tenant quotas API" do
+  let(:tenant) { FactoryGirl.create(:tenant) }
```

Comment:
Are you sure the quotas were created? Those `let`s are lazily evaluated, so they will only be created the first time you use one. You'd need to use `let!` instead. But personally I'm not a fan of doing that - I liked the way it was before. If they are an important detail for the running of the test, I prefer to see them in the body of the test.